backend/plonk/bn254/solidity.go

```diff
@@ -222,6 +223,17 @@ contract PlonkVerifier {
         revert(ptError, 0x64)
       }
 
+      /// Called when an operation on Bn254 fails
+      /// @dev for instance when calling EcMul on a point not on Bn254.
```

Comment:
Comment invalid? The error is for mod exp, but comment is about EcMul?

Reply:
right it's a mistake